examples/standalone/benchmarks/run_on_daint.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+## Arguments:
+# $1: number of timesteps to run
+# $2: number of ranks to execute with (ensure that this is compatible with fv3core)
+# $3: backend to use in gt4py
+# $4: target directory to store the output in
+# $5: path to the data directory that should be run
+#############################################
+# Example syntax:
+# ./run_on_daint.sh 60 6 gtx86
+
+exitError()
+{
+    echo "ERROR $1: $3" 1>&2
+    echo "ERROR     LOCATION=$0" 1>&2
+    echo "ERROR     LINE=$2" 1>&2
+    exit $1
+}
+
+SCRIPT=`realpath $0`
+SCRIPTPATH=`dirname $SCRIPT`
+ROOT_DIR="$(dirname "$(dirname "$(dirname "$SCRIPTPATH")")")"
+echo "scriptpath is $SCRIPTPATH"
+echo "root dir is : $ROOT_DIR"
+
+
+# check sanity of environment
+test -n "$1" || exitError 1001 ${LINENO} "must pass a number of timesteps"
+timesteps="$1"
+test -n "$2" || exitError 1002 ${LINENO} "must pass a number of ranks"
+ranks="$2"
+
+backend="$3"
+if [ -z "$3" ]
+  then
+    backend="numpy"
+fi
+
+target_dir="$4"
+if [ -z "$4" ]
+  then
+    target_dir="$ROOT_DIR"
+fi
+
+data_path="$5"
+if [ -z "$5" ]
+  then
+    data_path="/project/s1053/fv3core_serialized_test_data/7.0.0/c12_6ranks_standard/"
+fi
+
+
+# set up the virtual environment
+cd $ROOT_DIR
+rm -rf vcm_1.0
+
+echo "copying in the venv"
+cp -r /project/s1053/install/venv/vcm_1.0/ .
+git submodule update --init --recursive
+echo "install requirements"
+vcm_1.0/bin/python -m pip install external/fv3gfs-util/
+vcm_1.0/bin/python -m pip install .
+vcm_1.0/bin/python -m pip install gitpython
+
+# set up the experiment data
+cp -r $data_path test_data
+tar -xf test_data/dat_files.tar.gz -C test_data
+cp test_data/*.yml test_data/input.yml
```

Comment:
the data is not already expanded?

Reply:
no, it is still in the archive

Reply:
Since it is not a huge number of files, does it make sense to just keep the expanded version and run from that? This in theory takes up less space though, so that's good.